src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -244,10 +245,14 @@ const BrokerFormContent: FC<Props> = ({
               {status => (
                 <Input
                   type={InputType.Number}
-                  placeholder="1883"
+                  placeholder={`${
+                    !formContent || isNaN(formContent.brokerPort)
+                      ? DEFAULT_PORT
+                      : formContent.brokerPort
+                  }`}
                   name="port"
                   autoFocus={false}
-                  value={formContent.brokerPort}
+                  value={formContent.brokerPort ?? DEFAULT_PORT}
```

Comment:
this only works because `brokerPort` defaults to `undefined` instead of empty string. Can we change it to the following in case we ever change the default brokerPort value?

```suggestion
                  value={formContent.brokerPort ? formContent.brokerPort : DEFAULT_PORT}
```

Reply:
Same as 
```ts
formContent.brokerPort || DEFAULT_PORT
```

Reply:
That's fair @subirjolly I think we should just remove this ternary altogether